plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -1423,4 +1581,15 @@ class FlingGestureTest(
     const val FLING_DISPLACEMENT = 1000.0
     const val FLING_VELOCITY = 10000f
   }
-}
\ No newline at end of file
+}
+
+internal fun verifyNo(
+  ordering: Ordering = Ordering.UNORDERED,
+  timeout: Long = 0,
+  verifyBlock: MockKVerificationScope.() -> Unit
+) = verify(
+  ordering = ordering,
+  exactly = 0,
+  timeout = timeout,
+  verifyBlock = verifyBlock,
+)
```

Comment:
I'd rather either move this to some separate common test module and replace across the whole codebase or revert here back to `verify(exactly = 0)` to make those tests consistent with all others

Reply:
Yeah, can't use veiryfoNo util from the sdk tests unfortunately, tried to introduce here, but ok to use exactly = 0 until extenstion will be imported